src/Workspaces/Core/Portable/FindSymbols/FindReferences/BaseTypeFinder.cs

```diff
@@ -22,17 +21,14 @@ internal static partial class BaseTypeFinder
             ISymbol symbol, Project project, CancellationToken cancellationToken)
         {
             var solution = project.Solution;
-            var baseClassesAndInterfaces = FindBaseTypesAndInterfaces(symbol.ContainingType);
+            var baseClasses = FindBaseTypesAndInterfaces(symbol.ContainingType)
+                .WhereAsArray(t => t.TypeKind == TypeKind.Class);
```

Comment:
i believe there's just a FindBaseTypes right?  so you wouldn't need to filter?  But that said, i'm not sure i undestand the logic of what's going on here.  can you help clarify? (note: i'm asking for docs in the code).